sdk/textanalytics/test-resources.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "THe base resource name."
+            }
+        },
+        "testApplicationOid": {
+            "type": "String",
+            "metadata": {
+                "description": "The principal to assign the role to. This is application object id."
```

Comment:
nit: be consistent about using ID or id.

Reply:
updated throughout to use id.